src/js/msp/MSPHelper.js

```
@@ -2443,147 +2443,81 @@ MspHelper.prototype.setRawRx = function (channels) {
-                    } else if (dataCompressionType == 1) {
-                        // Read compressed char count to avoid decoding stray bit sequences as bytes
-                        const compressedCharCount = response.data.readU16();
+MspHelper.prototype.dataflashRead = function(address, blockSize, onDataCallback) {
+    let outData = [
+        address & 0xFF,
+        (address >> 8) & 0xFF,
+        (address >> 16) & 0xFF,
+        (address >> 24) & 0xFF,
+        blockSize & 0xFF,
+        (blockSize >> 8) & 0xFF,
+        1 // allow compression
```

ESLint

[error] Missing trailing comma.

(comma-dangle)